LEGO1/lego/sources/misc/legoutil.h

```diff
@@ -39,6 +39,9 @@ inline void Swap(T& p_t1, T& p_t2)
 	p_t2 = t;
 }
 
+// TEMPLATE: BETA10 0x10073c20
+// Swap
```

Comment:
Not sure if that annotation is correct. There is only one instance of `Swap`, but other examples suggest that the type is not part of the symbol for these templated functions. I think all of them will just be called `Swap`.